_includes/markdown/Design.md

```diff
@@ -171,12 +171,16 @@ If a management UI is impossible due to the nature of the project, credentials s
 <?php
 // Production API keys should ideally be defined in wp-config.php
 // This section should default to a development or noop key instead.
-if ( ! defined( 'CLIENT_MANDRILL_API_KEY' ) && ! ENV_DEVELOPMENT ) {
+if ( ! defined( 'CLIENT_MANDRILL_API_KEY' ) && defined( 'WP_ENVIRONMENT_TYPE' ) && ! in_array( wp_get_environment_type(), array( 'local', 'development' ) ) ) {
```

Comment:
@jamesmorrison - I think we should be more restrictive and only include the keys when it's production, instead of avoiding it on locals.

```
if ( ! defined( 'CLIENT_MANDRILL_API_KEY' ) {
    if ( defined( 'WP_ENVIRONMENT_TYPE' ) && 'production' === wp_get_environment_type() ) {
        define( 'CLIENT_MANDRILL_API_KEY', '1234567890' );
    }
}
```

My reasoning for this is that staging/preprod environments would include production creds otherwise.

Reply:
Also, you don't need to check for `defined( 'WP_ENVIRONMENT_TYPE' )`.

`wp_get_environment_type()` does this for you, and returns `''` if it's not set.

Reply:
I've updated the example per your suggestion, in https://github.com/10up/Engineering-Best-Practices/pull/378/commits/7f88bcb24f3e1bbd8a4e7875a6e60eaae6bdd6f3

Let me know if you think this should be expanded on - for example to include examples for other environments, or via a `switch` statement (based on the value returned from `wp_get_environment_type`).

Reply:
@gthayer This has been updated.